ipi_tests/examples/exampletools.py

```diff
@@ -30,6 +31,9 @@ def get_driver_info(
     socket_mode="unix",
     port_number=33333,
     address_name="localhost",
+    flags1=0.0,
```

Comment:
It would be better to make this more general, i.e. not enforce a fixed number of flags.

Reply:
see general comment below